app/services/csv/lettings_log_csv_service.rb

```diff
@@ -51,7 +51,13 @@ def set_csv_attributes
 
     def ordered_form_questions
       downloaded_form_years = LettingsLog.all.map(&:collection_start_year).uniq.compact
-      downloaded_form_fields = downloaded_form_years.count == 1 && downloaded_form_years[0].present? ? FormHandler.instance.get_form("#{downloaded_form_years[0]}_#{downloaded_form_years[0] + 1}").questions : FormHandler.instance.forms.first.second.questions
+
+      if downloaded_form_years.count == 1 && downloaded_form_years[0].present?
+        form_name = FormHandler.instance.form_name_from_start_year(downloaded_form_years[0], "lettings")
+        downloaded_form_fields = FormHandler.instance.get_form(form_name).questions
+      else
+        downloaded_form_fields = FormHandler.instance.current_lettings_form.questions
```

Comment:
I wonder if this needs to consider the case where a question has been dropped from the current form but is still there on the previous. You'd expect it to still be there in the download for those logs I think? In which case we might need a combined question set? cc. @CheshireSwift

Reply:
So I think this is only used for ordering, the field should still get downloaded as long as it is saved as a database field. It might just be at the end